### DISCUSSION TITLE
How can I rank my npm packages on Google?

### DISCUSSION THREAD

**Discussion by chufarligata** [npm/feedback#897] [General]
Hi, all

I'm begginner and really exited about coding. I was wondering how can I rank my npm packages on Google, so others can test it? I've noticed that there a lot of npm packages on Google, but my don't get indexed by Google. The settings are public access and the packages can be found in the npm registry. Thank you!

**Comment by monishcm** [comment]
Sorry, unfortunately we do not have information on how Google ranks its search results.
